docs/en/Community-Articles/2023-09-23-CMS-Kit-Demo/POST.md

```diff
@@ -0,0 +1,91 @@
+
+One of the standing out features of the CMS Kit is its [Menu System](https://docs.abp.io/en/abp/latest/Modules/Cms-Kit/Menus), which allows for the creation and dynamic ordering of application menu items. Say goodbye to static menus; with CMS Kit, you have the power to tailor your menu structure according to your evolving content needs.
+
+You can see the homepage of the application in the following figure:
+
+![](homepage.png)
+
+The application menu items in the navbar are **created & ordered dynamically** with the [CMS Kit's Menu System](https://docs.abp.io/en/abp/latest/Modules/Cms-Kit/Menus):
+
+![](menu-admin-side.jpg)
+
+### Custom Implementations with Comment & Reaction Features
+
+Our demo application goes a step further by demonstrating custom implementations, such as an **image gallery**, seamlessly integrated with CMS Kit's [Comment](https://docs.abp.io/en/abp/latest/Modules/Cms-Kit/Comments) & [Reaction](https://docs.abp.io/en/abp/latest/Modules/Cms-Kit/Reactions) Features:
+
+| Gallery | Detail Page  |
+|------------------------ |-----------------------|
+| ![](image-gallery.jpg) | ![](image-gallery-detail.jpg)  |
+
+It's pretty easy to integrate CMS Kit Features such as Comments & Reactions into your existing pages. You can check the [source code of the application](https://github.com/abpframework/cms-kit-demo/blob/main/src/CmsKitDemo/Pages/Gallery/Detail.cshtml) and see how to integrate the features.
+
+### Robust Blogging Capabilities
+
+Blogging has never been easier! With the CMS Kit's [Blogging Feature](https://docs.abp.io/en/abp/latest/Modules/Cms-Kit/Blogging), you can effortlessly manage your blog content, complete with [Ratings](https://docs.abp.io/en/abp/latest/Modules/Cms-Kit/Ratings), [Comments](https://docs.abp.io/en/abp/latest/Modules/Cms-Kit/Comments), [Tags](https://docs.abp.io/en/abp/latest/Modules/Cms-Kit/Tags), and [Reactions](https://docs.abp.io/en/abp/latest/Modules/Cms-Kit/Reactions) features as enabled:
+
+| Blog | Blog Post  |
+|------------------------ |-----------------------|
+| ![](blogs.jpg) | ![](blog-detail.jpg)  |
+
+You can enable/disable CMS Kit Features per blog on the admin side easily:
+
+![](cmskit-module-features.png)
+
+### Dynamic Pages with Style and Script Integration
+
+*Products* pages showcase the flexibility of the CMS Kit's [Pages Feature](https://docs.abp.io/en/abp/latest/Modules/Cms-Kit/Pages), allowing for dynamic content creation, style customization, and script integration. Your website can now truly reflect your unique brand and content style.
+
+You can create pages with dynamic URLs on the admin side:
+
+![](pages-admin-side.jpg)
+
+After you have created the page, you can access it via `/{slug}` URL on the public-web side:
+
+![](products-abp-commercial.png)
+
+## What's Next?
+
+Please try the CMS Kit Module now and provide [feedback](https://github.com/abpframework/abp) to help us to build a more effective content management kit!
```

Comment:
We can add a resources section with the following links:

* demo source code link
* demo lşive link
* cms kit document link
* cms kit pro document link